cli/command/base_test.go

```diff
@@ -0,0 +1,74 @@
+package command
+
+import (
+	"flag"
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	"github.com/quintilesims/layer0/api/job"
+	"github.com/quintilesims/layer0/cli/printer"
+	"github.com/quintilesims/layer0/cli/resolver/mock_resolver"
+	"github.com/quintilesims/layer0/client/mock_client"
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/urfave/cli"
+)
+
+type TestCommandBase struct {
+	client   *mock_client.MockClient
+	printer  *printer.TestPrinter
+	resolver *mock_resolver.MockResolver
+}
+
+func newTestCommand(t *testing.T) (*TestCommandBase, *gomock.Controller) {
+	ctrl := gomock.NewController(t)
+
+	tc := &TestCommandBase{
+		client:   mock_client.NewMockClient(ctrl),
+		printer:  &printer.TestPrinter{},
+		resolver: mock_resolver.NewMockResolver(ctrl),
+	}
+
+	return tc, ctrl
+}
+
+func NewJob() *models.Job {
```

Comment:
I don't think this is necessary. Seems like this will add "magic" to tests when `"job_id"` gets thrown around it's hard to tell where it came from

Reply:
Updated to:
```
job := &models.Job{
    JobID:  "job-id",
    Status: job.Completed.String(),
    Result: "entity-id",
}
```